src/main/java/com/mazyavr/schedule/service/TaskService.java

```diff
@@ -84,7 +84,7 @@ public Iterable<TaskEntity> getToday(long userId) {
     List<Long> ids = new ArrayList<>();
   
     for (TaskEntity t : taskRepository.findAll()) {
-      if (t.getProject().getUser().getId()==userId&&t.getStart().compareTo(time.plusDays(1)) < 0 && t.getEnd().compareTo(time) >= 0 && !t.isStatus()) {
+      if (t.getId()==userId&&t.getStart().compareTo(time.plusDays(1)) < 0 && t.getEnd().compareTo(time) >= 0 && !t.isStatus()) {
```

Comment:
Зачем это изменение? Надо сравнивать айди пользователя, а `t.getId()` это айди таски а не пользователя.

```suggestion
      if (t.getProject().getUser().getId() == userId && t.getStart().compareTo(time.plusDays(1)) < 0 && t.getEnd().compareTo(time) >= 0 && !t.isStatus()) {
```